### ISSUE TITLE
Push mode not working

### ISSUE THREAD

**Issue by arkon** [arkon/ng-sidebar#93]
@jraaaaanger reported this in #79:

> The content inside `ng-sidebar-container` is pushed over properly but the sidebar itself doesn't move at all, not totally sure why. `[mode]="'slide'"` works as expected.

**Comment by arkon** [comment]
@jraaaaanger Can I get a snippet of your template to see how it's roughly laid out? If you have any styles related to the sidebar, that'd be worth seeing as well.

**Comment by jraaaaanger** [comment]
@arkon I've been playing around and stripped it down to a simple sidebar with no styles at all, this is what I've got in my sandbox: 
```
<ng-sidebar-container>
  <ng-sidebar
    #sidebar
    [mode]="'push'">
    <p>Hello, World!</p>
  </ng-sidebar>
  <button (click)="sidebar.open()">OPEN</button>
  <button (click)="sidebar.close()">CLOSE</button>
  <p>A bunch of lorem ipsum here...</p>
</ng-sidebar-container>
```

This moves the buttons over and properly resizes the other stuff in the `ng-sidebar-container`, but the sidebar itself stays outside the view window. If I switch to `[mode]="'slide'"`, it works as expected. I've tried playing around with `dock` and `dockedSize` too, and those dock as expected, but it still doesn't move on `.open()`. 

Thanks for looking into this!

**Comment by arkon** [comment]
Ah, I figured it out. It's apparently mandatory to have `[(opened)]="someBooleanVariable"` for it to work. I'll fix that.

**Comment by jraaaaanger** [comment]
Nice! Ya that's working for me, thanks a ton!

**Comment by arkon** [comment]
Fixed in v5.0.3.
